docs/src/api/class-keyboard.md

```diff
@@ -172,6 +172,16 @@ Modifier keys DO influence `keyboard.down`. Holding down `Shift` will type the t
 
 Name of the key to press or a character to generate, such as `ArrowLeft` or `a`.
 
+### option: Keyboard.down.key
```

Comment:
Call this `eventKey` instead, to avoid confusion between key and key?

Reply:
Sure